library/utils.css

```diff
@@ -47,7 +47,7 @@
 .xs-text {
   line-height: 2rem;
   font-size: 0.75rem;
```

Comment:
can be used px

Reply:
Maintaing consistency with decreasing font-size.